packages/larva/scripts/larva/README.md

```diff
@@ -4,54 +4,26 @@ After this directory has been copied into a consuming project, [follow these ste
 
 # {Project} Larva
 
+This is configuration only instance of Larva. There should be no CSS or JS compiled here, only JSON configuration using existing styles from larva.css. If additional styles are needed, they should be [contributed upstream to the Larva packages](https://confluence.pmcdev.io/x/EQNCB).
+
 ## Links
 
 * [{Project} design tokens]().
-* [{Project} style guide on Invision]().
-* [{Project} style guide built on Larva server]().
+* [{Project} style guide on Invision or XD]().
+* [{Project} style guide on the static site]().
 
 ## Development
 
 To start developing:
 
-1. Run `npm install` from assets.
+1. Run `npm install` from assets/.
```

Comment:
Is this change intentional?

Reply:
Oh wait, I think it is supposed to indicate a file path...